docs/geedocs/answer/4485225.html

```diff
@@ -224,40 +202,32 @@ <h3 id="building_the_globe">Building the globe</h3>
-      <p>All of the globes present in the globe directory are listed and available for serving.</p>
+    <li>To start serving run the following command in the directory you unpacked Portable Server:
+      <p><code>python portable_server.py</code></p>
     </li>
-    <li>Click <strong>start serving</strong> next to the globe you wish to serve.</li>
+    <li>Navigate to the Portable Server administrative page at <code>http://localhost:9335</code>, using whatever <code>port</code> you've used in <code>portable.cfg</code>.</li>
   </ol>
-  <p>To stop serving a globe, return to the administration page at <code>http://localhost:9335</code>. The currently-served globe is listed in the first section; click <strong>stop serving</strong> to stop serving that globe.</p>
+  <p>You can select what globe is being served from the administrative page. Click on the folder icon (<img class="inline" alt="Portable folder icon" src="../art/fusion/portable/portable_folder_icon.png"/>), and you should a see a screen where you can select what globe, or map should be served by Portable Server.</p>
+  <p>You can shut down Portable Server from the administrative page, as well. Click on the hamburger menu button (<img class="inline" alt="Portable menu icon" src="../art/fusion/portable/portable_menu_icon.png"/>), and then on 'Quit'.</p>
   <hr/>
   <h3 id="broadcasting_a_globe">Broadcasting a globe</h3>
   <p>To share a globe with others on your network:</p>
   <ol class="spaced-list">
-    <li>If you're not already serving a globe, double-click the globe file, or select the globe to serve from the Portable Server interface.</li>
-    <li>In the <strong>Currently serving</strong> section of the Portable Server interface, click <strong>broadcast</strong>.</li>
-    <li>Enter a <strong>Key</strong> value and click <strong>Set Key</strong>. This key is simply an access key; it is not an encryption key.</li>
-    <li>In the <strong>Remote servers</strong> section, click <strong>add</strong>.</li>
-    <li>The <strong>Currently serving</strong> section now shows that you are <strong>Broadcasting</strong>. Click <strong>glr</strong> to create a connection file for this globe.
-      <p><img alt="Portable Globe" id="portable_glr" name="portable_glr" src="../art/fusion/portable/portable_glr.png"/></p>
-    </li>
-    <li>A <code>share.glr</code> file is created and saved to your browser's default download directory. This file can be sent to any user who requires a connection to your globe; the remote client must also be installed on their machine.</li>
+    <li>Make sure <code>disable_broadcasting</code> is set to <code>False</code> in your <code>portable.cfg</code>.</li>
+    <li>If you're not already serving a globe, select the globe to serve from the Portable Server interface.</li>
+    <li>On the Portable Server administrative page, click the broadcast icon (<img class="inline line-sized" alt="Broadcast globe icon" src="../art/fusion/portable/broadcast_false.gif"/>).</li>
```

Comment:
Yeah, the icon changes from `broadcast_false` to `broadcast_true` when you click it. If you click the `broadcast_true` one, you'd be stopping the broadcasting.

I didn't use `broadcast_true.gif` anywhere, but I thought, if we have one, we should have the other, as well, in case we need to use it later on when we update the documentation.